Source/Fuse.Reactive.JavaScript/ArrayMirror.uno

```diff
@@ -27,6 +27,9 @@ namespace Fuse.Reactive
 
 		public override void Unsubscribe()
 		{
+			if (_isDisposed) return;
+			_isDisposed = true;
```

Comment:
This feels a bit like abuse. Unsubscribe and Dispose isn't quite the same... Perhaps they should be, though?